microsoft_intune/icon_microsoft_intune/util/api.py

```diff
@@ -35,6 +36,30 @@ def wipe_managed_device(
     def managed_device_action(self, device_id: str, action: str):
         return self._call_api("POST", f"deviceManagement/managedDevices/{device_id}/{action}")
 
+    def get_managed_app(self, uuid: str) -> list:
+        return [self._call_api("GET", f"deviceAppManagement/mobileApps/{uuid}")]
+
+    @staticmethod
+    def filter_managed_apps_result(response, app_filter) -> list:
+        return list(filter(lambda iter_app: iter_app["displayName"].lower() == app_filter.lower(), response["value"]))
```

Comment:
```suggestion
        return list(filter(lambda iter_app: iter_app["displayName"].lower() == app_filter.lower(), response["value"]))
```
The indices lookups here may crash if they return None